library/classes/InsuranceNumbers.class.php

```diff
@@ -21,12 +21,13 @@ class InsuranceNumbers extends ORDataObject
                                                         "1D" => "Medicaid Provider Number","1G" => "Provider UPIN Number","1H" => "Champus Identification Number","1J" => "Facility ID Number",
```

Comment:
if this is not part of the PR topic, then revert this script. easiest way to do this is to simply replace this file from the file in master on github:
wget https://raw.githubusercontent.com/openemr/openemr/master/library/classes/InsuranceNumbers.class.php

Reply:
many thanks brady, i've made the changes and will do the PR very shortly